### PR TITLE
Fix small typos and format style

### PR DESCRIPTION
**Pull request by rgaiacs**
@apawlik Can you review and merge?

### CODE REVIEW

File: _posts/2016/09/2016-09-28-attendance-nz.md [hunk 1]
```diff
@@ -11,49 +11,41 @@ Having taught and helped at a series of workshop over the past few months Tom Ke
 
 Please note that these are the authors' views and thus they should not be treated as representative for their home institutions.
 
-Tom based his opinions after having  taught at various workshops in Australia in New Zealand: <br/>
+Tom based his opinions after having taught at various workshops in Australia and in New Zealand, including Research Bazaar 2015 at the [University of Melbourne](http://unimelb.edu.au), Research Bazzar 2016 at the [University of Otago](http://www.otago.ac.nz/), University of Otago, [University of Canterbury](http://www.canterbury.ac.nz/), and [NeSI](https://www.nesi.org.nz/). There have been several other workshops in New Zealand facilitated by [NeSI](https://www.nesi.org.nz/), in Auckland, Wellington, and Palmerston North over the past year.
```
Comment (rgaiacs) [inline _posts/2016/09/2016-09-28-attendance-nz.md:14]: The list as a little strange to me. I can undo this if you want.

File: _posts/2016/09/2016-09-28-attendance-nz.md [hunk 2]
```diff
@@ -11,49 +11,41 @@ Having taught and helped at a series of workshop over the past few months Tom Ke
 
 Please note that these are the authors' views and thus they should not be treated as representative for their home institutions.
 
-Tom based his opinions after having  taught at various workshops in Australia in New Zealand: <br/>
+Tom based his opinions after having taught at various workshops in Australia and in New Zealand, including Research Bazaar 2015 at the [University of Melbourne](http://unimelb.edu.au), Research Bazzar 2016 at the [University of Otago](http://www.otago.ac.nz/), University of Otago, [University of Canterbury](http://www.canterbury.ac.nz/), and [NeSI](https://www.nesi.org.nz/). There have been several other workshops in New Zealand facilitated by [NeSI](https://www.nesi.org.nz/), in Auckland, Wellington, and Palmerston North over the past year. 
 
-* Research Bazaar 2015 at the [University of Melbourne](http://unimelb.edu.au) and 2016 at the [University of Otago](http://www.otago.ac.nz/);
 
-* Several Software Carpentry workshops organised by the University of Otago, [University of Canterbury](http://www.canterbury.ac.nz/), and [NeSI](https://www.nesi.org.nz/), in February 2015, June 2015, and June 2016;
+### Main point
```
Comment (rgaiacs) [inline _posts/2016/09/2016-09-28-attendance-nz.md:17]: GitHub Pages doesn't threat headers without space after `#` as headers. :-( Is good to keep the same style here as well.

File: _posts/2016/09/2016-09-28-attendance-nz.md [hunk 3]
```diff
@@ -11,49 +11,41 @@ Having taught and helped at a series of workshop over the past few months Tom Ke
...
 
-* A Data Carpentry workshop run at the University of Otago in February 2016.
+Over the course of several workshops we've had relatively minor problems with "no-shows" (people signing up and not attending) or "drop-offs" (people not returning for future days or sessions). However, in the case of the oversubscribed workshops it was still somewhat frustrating. This has led to discussions about how we may address the issues related to the attendance to ensure that others who would have attended for the entire workshop, but ended up on the waiting list, do not miss out on places.
```
Comment (rgaiacs) [inline _posts/2016/09/2016-09-28-attendance-nz.md:19]: Typo: overscubscribed -> oversubscribed

File: _posts/2016/09/2016-09-28-attendance-nz.md [hunk 4]
```diff
@@ -11,49 +11,41 @@ Having taught and helped at a series of workshop over the past few months Tom Ke
...
 
+### Approach 1: Registration Fee
 
-###Approach 1: Registration Fee
-
-We discussed further no-shows with Bill Mills when we were doing ResBaz/SWC in Melbourne and Christchurch. He mentioned a soultion suggested by Software Carpentry of appying a small registration fee to make sure those who register actually attend or cancel giving the organisers some notice. Based on the experiences of several hosts accross  this usually results in no-show numbers dropping to below 5%.
+We discussed further no-shows with Bill Mills when we were doing ResBaz/SWC in Melbourne and Christchurch. He mentioned a solution suggested by Software Carpentry of applying a small registration fee to make sure those who register actually attend or cancel giving the organisers some notice. Based on the experiences of several hosts accross  this usually results in no-show numbers dropping to below 5%.
```
Comment (rgaiacs) [inline _posts/2016/09/2016-09-28-attendance-nz.md:34]: Typo: appying -> applying

File: _posts/2016/09/2016-09-28-attendance-nz.md (hunk 1 above)
Comment (gvwilson) [reply _posts/2016/09/2016-09-28-attendance-nz.md:14]: Duplicates "University of Otago".

**Pull request merged**